_about/notes/index.md

```diff
@@ -16,17 +16,16 @@ toc: false
 
 {% include section-index.html docs=site.about %}
 
+The [latest](https://github.com/istio/istio/releases) Istio monthly release is {{site.data.istio.version}} which
+you can download with:
 
-- The [latest](https://github.com/istio/istio/releases) Istio monthly release is {{site.data.istio.version}}. It is downloaded when the following is used(*):
-  ```
-  curl -L https://git.io/getLatestIstio | sh -
-  ```
+```
+curl -L https://git.io/getLatestIstio | sh -
+```
 
-- The most recent 'stable' release is [0.2.12](https://github.com/istio/istio/releases/tag/0.2.12), the matching docs are [archive.istio.io/v0.2/docs/](https://archive.istio.io/v0.2/docs/)
-  ```
-  curl -L https://git.io/getIstio | sh -
-  ```
+The most recent stable release is [0.2.12](https://github.com/istio/istio/releases/tag/0.2.12). You can
+find the docs for that release [here](https://archive.istio.io/v0.2/). You can download 0.2.12 with:
```

Comment:
using "here" for links is not the most useful kind of links
http://stephanieleary.com/2015/05/why-click-here-is-a-terrible-link-and-what-to-write-instead/

Reply:
I improved the links. Thanks for the article pointer, I'm going to add that link to our style guide.

Reply:
well you removed the the doc link entirely... I will put it back (and the security warning) in a follow up PR

Reply:
edit: oh nvm it's at the bottom now, cool